src/sessions/screen-recording-session-handler.ts

```diff
@@ -52,10 +58,31 @@ export default class ScreenRecordingSessionHandler extends BaseSessionHandler {
 
     await addMediaPromise;
 
-    return super.acceptSession(session, params);
+    await super.acceptSession(session, params);
+    await this.updateScreenRecordingMetadatas(session, params.screenRecordingMetadatas);
+
   }
 
   async endSession (conversationId: string, session: IExtendedMediaSession, reason?: Constants.JingleReasonCondition): Promise<void> {
     throw createAndEmitSdkError.call(this.sdk, SdkErrorTypes.not_supported, `sessionType ${this.sessionType} must be ended remotely`, { conversationId });
   }
+
+  updateOutgoingMedia (session: IExtendedMediaSession, options: IUpdateOutgoingMedia): never {
+    this.log('warn', 'Cannot update outgoing media for screen recording sessions', { sessionId: session.id, sessionType: session.sessionType });
```

Comment:
Can we really not update media for SR sessions?

Reply:
well, yes and no. We *could* but there's no precedent for that yet and when we do, we'd need to send new screen metadata.